src/test/java/com/olgaboiar/mint/loggers/FileLoggerTest.java

```diff
@@ -0,0 +1,49 @@
+package com.olgaboiar.mint.loggers;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class FileLoggerTest {
+    @Test
+    void getMethodReturnsNullWhenNoMethodSet() throws IOException {
```

Comment:
Similar to: https://github.com/olgaboiar/MintHTTPServer/pull/6/files#r252822342

Reply:
renamed in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/eeef1cc27cafdd69873c1ff48e4fe135e672db2c